canonical/ubuntu.com
# Single CVE page

## Done

- Applied redesign as per [mock-up](https://www.figma.com/file/nUJEBpOPTW74RqDRuMQXxB/23.04-U.com---CVEs?node-id=949%3A23161&mode=dev)

## Future work

The JS functionality to toggle unmantained releases will be handled in a separate pr, as will further styling updates to the status table. "Other references" will also be handled separately as this field needs additional refactoring to satisfy the design.

## QA

- View the site locally in your web browser at: 
    - https://ubuntu-com-13679.demos.haus/security/CVE-2022-28737
    - https://ubuntu-com-13679.demos.haus/security/CVE-2023-5678
    - https://ubuntu-com-13679.demos.haus/security/CVE-2011-1497
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against design, noting known discrepancies listed in future work above 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5648
